mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -271,6 +260,20 @@ def _rpc(self, method, *args):
             raise ProtocolError(ProtocolError.MISMATCHED_API_ID)
         return result['result']
 
+    def _is_app_running(self):
```

Comment:
The name of this function kinda indicates that we could call this at any point to check if the app  is running. Is this true? Would attempting another connection when there's an ongoing connection mess things up?